egs2/TEMPLATE/asr1/scripts/utils/upload_models_to_hub.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+rm model_names.txt
+echo "$1"
+model_name="$1"
+echo ${model_name}
+
+python get_model_names.py "${model_name}" >> model_names.txt # Saves model path in a file
+
+IFS=$'\n' GLOBIGNORE='*' command eval  'a=($(cat model_names.txt))'
+
+for value in ${a[@]}
+do
+
+echo ${value} # This is model path
+
+
+escapeString="_"
+pattern1="/"
+pattern2="+"
+pattern3=" " # Repo name does not accept /
+model_name=${model_name//${pattern3}/${escapeString}} 
+
+repo_name=${model_name}
+repo_name=${repo_name//${pattern1}/${escapeString}} 
+repo_name=${repo_name//${pattern2}/${escapeString}} 
+# Get name of hugging face repo
+
+rm -rf dest/*
+unzip  ${value} -d dest/ # Save data in dest folder
+
+# # Create hugging face repo and push all data to repo
+transformers-cli repo create ${repo_name}  --organization espnet
```

Comment:
I am not sure if people could freely create a repo using ESPnet as organization. It will be better to remove organization.

Reply:
I think it is better to ask people to be requested to add to ESPNet organisation and then upload models there. I believe all models are uploaded using the organisation name for standardisation.

Reply:
I think it is a good idea to add things to ESPNet organisation.